signer/migrations/0003__create_tables.sql

```diff
@@ -319,3 +319,30 @@ CREATE TABLE sbtc_signer.swept_deposits (
 -- unique per 'sweep_transaction_id'.
 CREATE UNIQUE INDEX uix_swept_deposits_req_txid_req_output_index_pkgd_txid
     ON sbtc_signer.swept_deposits(deposit_request_txid, deposit_request_output_index, sweep_transaction_txid);
+
+
+CREATE TABLE sbtc_signer.bitcoin_txs_sighashes (
+    txid BYTEA PRIMARY KEY,
+    chain_tip BYTEA NOT NULL,
+    prevout_txid BYTEA NOT NULL,
+    prevout_output_index INTEGER NOT NULL,
+    sighash BYTEA NOT NULL,
+    prevout_type sbtc_signer.prevout_type NOT NULL,
+    validation_result TEXT NOT NULL,
+    is_valid_tx BOOLEAN NOT NULL,
+    construction_version TEXT NOT NULL,
+    created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+CREATE TABLE sbtc_signer.bitcoin_withdrawals_outputs (
+    request_id BIGINT PRIMARY KEY,
```

Comment:
I noticed that wherever the withdrawal's request_id is used in ther other tables as a primary key, it is always paired with the block_hash. Not sure if it's required here, or if it's necessary to declare a foreign key to specific table

Reply:
It should be paired with the block hash because withdrawal requests are only unique that way. Reorgs on Stacks can lead to a withdrawal request with the same ID but where the contents of the request can be different from the earlier one.